worlds/jakanddaxter/Locations.py

```diff
@@ -0,0 +1,73 @@
+from BaseClasses import Location, CollectionState
+from .GameID import jak1_name
+from .locs import (OrbLocations as Orbs,
+                   CellLocations as Cells,
+                   ScoutLocations as Scouts,
+                   SpecialLocations as Specials,
+                   OrbCacheLocations as Caches)
+
+
+class JakAndDaxterLocation(Location):
+    game: str = jak1_name
+
+    # In AP 0.5.0, the base Location.can_reach function had its two boolean conditions swapped for a faster
+    # short-circuit for better performance. However, Jak seeds actually generate faster using the older method,
+    # which has been re-implemented below.
+    def can_reach(self, state: CollectionState) -> bool:
+        assert self.parent_region, "Can't reach location without region"
+        return self.parent_region.can_reach(state) and self.access_rule(state)
```

Comment:
As of https://github.com/ArchipelagoMW/Archipelago/pull/3724, core also made this change, so I would prefer the override to be removed now

Reply:
Oho, I was not aware. Thanks for the heads up!